.github/workflows/pull_request_release.yml

```diff
@@ -1,7 +1,10 @@
 name: One App Release Pull Request
 on:
-  schedule:
```

Comment:
Are we not automatically raising a PR every week?

Reply:
good catch this was accidentally commited

Reply:
update, thank you :-D